.github/workflows/ci.yml

```diff
@@ -8,6 +8,11 @@ jobs:
   ci:
     name: CI
     runs-on: ubuntu-latest
+    services:
+      mongo:
+        image: mongo:4.4-focal
```

Comment:
Note 📖 

Adjusted with the current MongoDB version spec declared in Docker compose config.

https://github.com/reearth/reearth-backend/blob/83f9b135f57e74d55361b0a51bd6ce1ce29b16c6/docker-compose.yml#L20